### PR TITLE
NDS-218 Alphabet component

### PR DESCRIPTION
**Pull request by barkertron**
https://nicedigital.atlassian.net/browse/NDS-218

### CODE REVIEW

File: components/nds-alphabet/nds-alphabet.d.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+		[key: string]: unknown;
+	}
+
+	export interface LetterProps {
+		children: React.ReactNode;
+		label?: string;
+		chunky?: boolean;
+		to?: string | false;
+		[key: string]: unknown;
+	}
```
Comment (ediblecode) [inline components/nds-alphabet/nds-alphabet.d.ts:17]: I think we need to export the definition for the component itself?

File: components/nds-alphabet/package.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    "url": "git+https://github.com/nice-digital/nice-design-system.git"
+  },
+  "scripts": {
+    "test": "echo \"Error: run tests from root\" && exit 1"
+  },
+  "bugs": {
+    "url": "https://github.com/nice-digital/nice-design-system/issues"
+  },
+  "keywords": [
+    "column",
```
Comment (ediblecode) [inline components/nds-alphabet/package.json:30]: These keywords need updating

File: components/nds-alphabet/src/__tests__/nds-alphabet.test.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+		const wrapper = shallow(<Alphabet>{letterElements}</Alphabet>);
+		expect(wrapper.children()).toHaveLength(allLetters.length);
+	});
+
+	it("should render chunky child components when the chunky variant is specified on the parent", () => {
+		const wrapper = shallow(<Alphabet chunky>{letterElements}</Alphabet>);
+		expect(
+			wrapper
+				.childAt(0)
+				.render() //  Used here due to this issue: https://github.com/enzymejs/enzyme/issues/1177#issuecomment-332717606
```
Comment (ediblecode) [inline components/nds-alphabet/src/__tests__/nds-alphabet.test.js:40]: Good comment, thanks

**Review by ediblecode — CHANGES_REQUESTED** [review]

File: components/nds-alphabet/nds-alphabet.d.ts (hunk 1 above)
Comment (barkertron) [reply components/nds-alphabet/nds-alphabet.d.ts:17]: Done!

File: components/nds-alphabet/package.json (hunk 2 above)
Comment (barkertron) [reply components/nds-alphabet/package.json]: Done!

**Pull request closed without merging**